.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8"]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: pip-${{ hashFiles('setup.cfg') }}-${{ hashFiles('tox.ini') }}
```

Comment:
I went with a compound hash here because we specify some dependencies in `setup.cfg` and some in `tox.ini`. `tox.ini` contains the Django version ranges that actually get used and the linting dependencies.